ui/analyse/src/study/studyCtrl.ts

```diff
@@ -250,6 +250,8 @@ export default class StudyCtrl {
     if (this.members.canContribute()) this.form.openIfNew();
 
     this.instanciateGamebookPlay();
+
+    window.addEventListener('popstate', () => window.location.reload());
```

Comment:
a full page reload? what is this about?

Reply:
That was previously in analyse/view.ts and it was something I added during early stages of the address bar url fixup work. I moved it to studyCtrl because it's related to the address bar fixup code there. At the time, it was needed to make sure the history was navigated properly when browser forward/back navigation occurred.